libraries/AP_Baro/AP_Baro.cpp

```diff
@@ -27,6 +27,7 @@
 #include <AP_BoardConfig/AP_BoardConfig.h>
 
 #include "AP_Baro_BMP085.h"
+#include "AP_Baro_BMP280.h"
```

Comment:
Is the bmp280 so different from the 085 that it needs its own file? Could it share the same file?